azurerm/internal/services/subscription/data_source_subscriptions.go

```diff
@@ -34,6 +34,11 @@ func dataSourceSubscriptions() *schema.Resource {
 				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"resource_manager_id": {
```

Comment:
can we make this just `id` to match other resources:

```suggestion
						"id": {
```